### ISSUE TITLE
RFE: Add unset

### ISSUE THREAD

**Issue by djerius** [uperl/Shell-Config-Generate#27]
Sometimes you need to delete an environment variable completely (just setting it to the empty string isn't enough).  It'd be great if one could do something like:

```
$config->unset( 'FOO' );
```
which would translate into `unset FOO`, ( or `unsetenv FOO` for csh/tcsh).

**Comment by plicease** [comment]
Agree, with the caveat that we need to support as well as possible all supported shells.  I believe `cmd.exe` and `command.com` actually do unset with `SET FOO=`, (but that is a limitation of those shells) and I am not sure about PowerShell.

**Comment by plicease** [comment]
https://www.itprotoday.com/powershell/powershell-one-liner-deleting-environment-variables

**Comment by plicease** [comment]
Implemented in #30

**Comment by djerius** [comment]
Thanks!

**Comment by plicease** [comment]
> Thanks!

sure.  sorry it took so long :)
